server.js

```diff
@@ -32,15 +66,9 @@ app.all('/profile-edit/server/whichrt', (req, res) => {
 
   if (reqUri !== null) {
     if (versoSpoof.ontologyUrls.includes(reqUri)) {
-      // FIXME:  there's probably a better way to find the value in array than forEach, but there are only 5 urls
-      let json
-
-      versoSpoof.owlOntUrl2JsonMappings.forEach((el) => {
-        if (reqUri === el.url) {
-          json = el.json
-        }
-      })
-      res.status(200).type('application/json').send(json)
+      const json = versoSpoof.owlOntUrl2JsonMappings.find(el => reqUri === el.url).json
+
+      res.json(json)
```

Comment:
can you say more about this line vs old line 43 ?   is status 200 the default?  Is application/json the default type?

Reply:
@ndushay 	Oh, shoot, I meant to leave a comment about this line but missed it. Sorry! Sure.

We can safely shorten `res.status(200).type('application/json').send(json)` to `res.json(json)` because Express will return an HTTP 200 OK by default, and calling `.json()` automatically sets the content type (to `application/json`), sends the content to the client, and ends the request/response cycle.